red-hat-storage/cephci
# CEPH-83571710: Tier-2 test to verify parallel IOPs on an object in a pool

[CEPH-83571710](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83571710): Tier-2 test to verify parallel I/O operations on the same object parallely from two different clients.

Test modules added - 
tests/rados/test_object_ops.py

Test suites modified -
- suites/pacific/rados/tier-2_rados_test_omap.yaml
- suites/quincy/rados/tier-2_rados_test_omap.yaml

Steps:
1. Create a replicated pool
2. Perform the below two tasks parallely:
      - Use rados put to write 4MB data to an object 15 times with
          initial offset 0 and incremental offset of 8MB from client node
      - Use rados put to write 4MB data to an object 15 times with
          initial offset 4MB and incremental offset of 8MB from installer node
3. Verify the ceph df stats and ensure the pool has only 1 object
    and stored data is 120 MB
4. Delete the created pool

Logs-
RHCS 5.3:
RHCS 6.1: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-8R3JF2

Signed-off-by: Harsh Kumar <hakumar@redhat.com>
